lms/djangoapps/teams/static/teams/js/collections/team.js

```diff
@@ -3,6 +3,8 @@
     define(['teams/js/collections/base', 'teams/js/models/team', 'gettext'],
         function(BaseCollection, TeamModel, gettext) {
             var TeamCollection = BaseCollection.extend({
+                sortField: 'name',
```

Comment:
Note that in this case we are planning to make the default sort different from what is in the API (which is alphabetical).